backend/orchestrator/resources/eventBridge.ts

```diff
@@ -0,0 +1,16 @@
+import { EventBus } from 'aws-cdk-lib/aws-events';
+import { Construct } from 'constructs';
+
+export class OrchestratorEventBus extends Construct {
+  public eventBusArn: string;
```

Comment:
It would be best to directly put the `eventBus` as an attribute, so that the instantiating construct can access anything inside it.

Same idea for the dynamo construct